src/layouts/default.html.handlebars

```diff
@@ -54,7 +67,7 @@
                     </div>
 
                 </article>
-            
+
             </div>
 
             <footer class="row infusion-docs-footer">
```

Comment:
Could the link text for the link to fluidproject.org be "the Fluid Project" instead of the URL?